tests/recovery.py

```diff
@@ -39,9 +39,9 @@ def check_nodes_have_msgs(nodes, txs):
     for node in nodes:
         with node.user_client() as c:
             for n, msg in txs.priv.items():
-                c.do("LOG_get", {"id": n}, {"msg": msg.encode()})
+                c.do("LOG_get", {"id": n}, None, {"msg": msg.encode()})
             for n, msg in txs.pub.items():
-                c.do("LOG_get_pub", {"id": n}, {"msg": msg.encode()})
+                c.do("LOG_get_pub", {"id": n}, None, {"msg": msg.encode()})
```

Comment:
As above, prefer `<method>, <params>, expected_arg=XXX` to `<method>, <params>, None, XXX`